src/main/java/browserview/ChromeDriverEx.java

```diff
@@ -50,9 +52,7 @@ public void get(String url, boolean isForceRefresh) throws WebDriverException {
             logger.info("Test loading page: " + url);
             testGet();
         } else {
-            if (!isForceRefresh && (driver.getCurrentUrl().equalsIgnoreCase(url) ||
-                driver.getCurrentUrl().equalsIgnoreCase(url + "/commits") ||
-                driver.getCurrentUrl().equalsIgnoreCase(url + "/files"))) {
+            if (!isForceRefresh && (driver.getCurrentUrl().equalsIgnoreCase(url))) {
```

Comment:
Makes sense to change `get` to what you do, but I don't think it's a good idea to overload it with the `BiPredicate`. Let me get back to you in a moment.